docusaurus/docs/protocol/primitives/probabilistic_proofs.md

```diff
@@ -1,23 +1,514 @@
+- **Success**:
+  - A dishonest Supplier gets caught (i.e. is required to provide a Proof and fails, resulting in a penalty)
+  - Taken from the network's perspective
+- **Failure**:
+  - A dishonest Supplier does not get caught (i.e. is not required to provide a Proof and receives rewards without providing actual service)
+  - An honest Supplier is rewarded
+  - All other outcomes
+  - Does not include _short-circuited_ (i.e. Claim.ComputeUnits > ProofRequirementThreshold)
+
+### Onchain Governance Parameters
+
+- **ProofRequestProbability (p)**: The probability that a Claim will require a Proof.
+- **Penalty (S)**: The amount of stake slashed when a Supplier fails to provide a required Proof.
+- **Reward per Claim (R)**: The reward received for a successful Claim without Proof.
+- **Maximum Claims Before Penalty (k)**: The expected number of false Claims a Supplier can make before getting caught.
+
+We note that `R` is variable and that `SupplierMinStake` is not taken into account in the definition of the problem.
+As will be demonstrated by the end of this document:
+
+- Reward per Claim (`R`) will be equal to the `ProofRequirementThreshold` (POKT)
+- Penalty (`S`) will be less than or equal to the `SupplierMinStake` (in POKT)
+
+### Dishonest Supplier: Calculating the Expected Value
+
+The dishonest Supplier's strategy:
+
+- Submit false Claims repeatedly, hoping not to be selected for Proof submission.
+- Accept that eventually, they will be caught and penalized.
+
+#### Modelling a Dishonest Supplier's Strategy using a Geometric PDF (Probability Distribution Function)
+
+The number of successful false Claims before getting caught follows a [Geometric distribution](https://en.wikipedia.org/wiki/Geometric_distribution):
+
+**Probability of Not Getting Caught (q)**:
+$$ q = 1 - p $$
+
+**Probability of Getting Caught on the `(k+1)`th Claim**:
+$$ P(X = k+1) = q^k \cdot p $$
+
+#### Expected Number of False Claims (Failures) Before Getting Caught (Success)
+
+$$ E[K] = \frac{q}{p} $$
+
+Recall:
+
+- **Failure**: The network does not catch a dishonest Supplier
+- **Success**: The network catches a dishonest Supplier
+
+#### Modelling a Dishonest Supplier's Strategy using a Geometric CDF (Cumulative Distribution Function)
+
+Above, we have been tracking the probability that `Pr(X=k+1)`, the probability
+of `k` failures (Supplier escapes without penalty) until a single success (Supplier)
+is penalized. This can be modeled using a Geometric PDF (Probability Distribution Function).
+
+In practice, we need to track the likelihood of `k or less` failures `Pr(X<=k)`,
+until a single success. This can be modeled using a Geometric CDF.
+
+To simplify the math, we'll be using the Expected Value of a Geometric PDF
+due to its [simpler proof formulation](https://en.wikipedia.org/wiki/Geometric_distribution#Proof_of_expected_value), guaranteeing the results be **AT LEAST**
+as secure when compared to the Geometric CDF.
```

Comment:
Not sure what you mean here, this is probably the rot of an issue I signal later.

Reply:
Aligned.

Reply:
@RawthiL So is the issue in my definition is the "simplification" I made because I couldn't find a closed form definition?

After reading and thinking through your comments (still an evolving thought), my understanding is that we need to compute `E[<=K]` everywhere but if you can prepare a quick notebook per your comment [here](https://github.com/pokt-network/poktroll/pull/899#discussion_r1835013327), it would be super helpful


<img width="923" alt="Screenshot 2024-11-11 at 12 35 47 PM" src="https://github.com/user-attachments/assets/c4e4a76e-8a4c-4bf3-a91c-8e7e7e5812b4">
<img width="1000" alt="Screenshot 2024-11-11 at 12 36 01 PM" src="https://github.com/user-attachments/assets/36a27929-9c20-465b-ac01-28a0da7c7577">
<img width="994" alt="Screenshot 2024-11-11 at 12 36 10 PM" src="https://github.com/user-attachments/assets/a5cef9db-67be-4e66-88f6-80b23c63e459">
<img width="887" alt="Screenshot 2024-11-11 at 12 36 21 PM" src="https://github.com/user-attachments/assets/7d9a0259-a4e0-40d4-b82c-35fb1eae9ecb">